### PR TITLE
Build one entry point for each command

### LINKED ISSUE #6: Use commander to define commands arguments and generate documentation
Have each script command register itself as a command in [commander](https://www.npmjs.com/package/commander), so it's possible to validate each argument and easily provide documentation.

### PR DESCRIPTION
**Pull request by facuspagnuolo**
**MERGE AFTER #55** (change base branch to the [refactor branch](https://github.com/zeppelinos/zos-cli/tree/refactor_branch) before merging)
Fixes #6 
Related to #9

This PR introduces a new entry point for each command. It allows us to use truffle just when needed and build custom helpers for each command.

### CODE REVIEW

File: bin/developers.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const initDistribution = require('../src/commands/developers/init-distribution')
+const deploy = require('../src/commands/developers/deploy')
+const register = require('../src/commands/developers/register')
+
+module.exports = function processDevelopersCommands(program) {
```
Comment (spalladino) [inline bin/developers.js]: I'd rename `process` to `register`, but it's a detail

File: src/commands/developers/init-distribution.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const initDistribution = require('../../scripts/init-distribution')
+
+module.exports = function(program) {
+  program
+    .command('init-distribution <name> <kernel>')
+    .description("Initialize your distribution project for standard libraries .\n  " +
+      "Provide a distribution <name> for your project\n  " +
+      "Provide the zeppelin_os <kernel> address where your releases are going to be published")
```
Comment (spalladino) [inline src/commands/developers/init-distribution.js]: Remember that backticks allow for [multiline strings](https://stackoverflow.com/a/805113/12791)

File: bin/zos-cli.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
 #! /usr/bin/env node
 
-const Path = require('path')
-const shelljs = require('shelljs')
-
-function run() {
-  const bin = 'npx truffle exec'
-  const script = Path.resolve(__dirname, '../src/entrypoint.js')
```
Comment (spalladino) [inline bin/zos-cli.js]: We should be able to remove the `sed` script from the prepackage script in package.json after this change, right?

File: bin/developers.js (hunk 1 above)
Comment (facuspagnuolo) [reply bin/developers.js]: agree!

File: bin/zos-cli.js (hunk 3 above)
Comment (facuspagnuolo) [reply bin/zos-cli.js]: definitely!

Comment (facuspagnuolo) [reply bin/zos-cli.js]: I addressed this comment here https://github.com/zeppelinos/zos-cli/pull/63

**Pull request merged**